karate-tests/src/test/java/patients/healthcareWorker/invalidHeaders/invalidHeaders.feature

```diff
@@ -64,3 +65,26 @@ Scenario Outline: x-request-id errors: patient <operation> - <diagnostics>
     | search       |                            | Invalid request with error - X-Request-ID header must be supplied to access this resource   | MISSING_VALUE   |
     | get          | 1234                       | Invalid value - '1234' in header 'X-Request-ID'                                             | INVALID_VALUE   |
     | search       | 1234                       | Invalid value - '1234' in header 'X-Request-ID'                                             | INVALID_VALUE   |
+
+  @sandbox-only
+  Scenario Outline: Auth errors: patient <operation> - <diagnostics> 
+    * def query = operation == 'search' ? { family: "Capon", gender: "male", birthdate: "eq1953-05-29" } : null
+    * def target = operation == 'search' ? 'Patient' : `Patient/${nhsNumber}`
+    
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * requestHeaders.authorization = headerValue
+    * configure headers = requestHeaders
+    
+    * path target
+    * params query 
+    * method get
+    * status 401
+    * assert utils.validateResponseHeaders(requestHeaders, responseHeaders) 
+    * def display = 'Access Denied - Unauthorised'
+    * def expectedResponse = read(`classpath:mocks/stubs/errorResponses/ACCESS_DENIED.json`)
+    * match response == expectedResponse
+  
+    Examples:
+      | operation    | headerValue                     | diagnostics                    |
+      | get          | Bearer                          | Missing access token           |
+      | get          | Bearer HEALTHCARE_WORKER        | Invalid Access Token           | 
```

Comment:
Sorry I think I asked this last time but I've forgotten. Are these scenarios being covered in the postman collection, if so when?

Reply:
Postman collection scenarios will be added under https://nhsd-jira.digital.nhs.uk/browse/SPINEDEM-2269

Reply:
Thanks, I'll make a note so I hopefully don't ask again. It might be worth adding to the ticket if you wanted to cover this in the scenario.